README.md

```diff
@@ -1,87 +1,71 @@
-AndroidUtils is a collection of util classes that I use in almost every project - I might as well give them their own module. This is more for personal use (I don't expect most people to use this as a dependency in their own projects, other than stealing a snippet or two), but feel free to add your own contributions as you wish.
+# HMOSUtils
 
-[![](https://jitpack.io/v/me.jfenn/AndroidUtils.svg)](https://jitpack.io/#me.jfenn/AndroidUtils)
+A HMOS Utils library is collection of utils classes.
```

Comment:
Please make it more descriptive as this lib is mainly created for SlideActionView.

Reply:
Done